### DISCUSSION TITLE
Custom buttons?

### DISCUSSION THREAD

**Discussion by dcastillo3** [the-maazu/react-material-music-player#32] [Ideas]
I would like to request custom buttons added to the player.

![custom_buttons](https://user-images.githubusercontent.com/10469758/193524599-72e80480-2ab0-4415-962e-9b9f3472bf94.jpg)
![custom_buttons_2](https://user-images.githubusercontent.com/10469758/193524602-4868d0d7-1cea-4235-b46f-3c52c668fa27.jpg)

**Comment by the-maazu** [comment]
Yes played around with the idea of offering more customisation options, but I also do not want to make bloatware. Keeping to essential features for now, so I advice others to make a fork of the project if they need to include extra features/buttons. Should you need help on your fork, I am available to help.
